.dockerignore

```diff
@@ -0,0 +1,9 @@
+# ignore the big files when building yourself
+**/db
+**/diffs
+**/*.log
+**/*.bin
+**/*.idx
+**/*.map
+**/*.osm.bz2
+README.md
```

Comment:
forgot about that. handy if you build yourself but already have all the data files in the project directory. else docker would initially load all that data to context.

Reply:
Inded, might be handy, but as .dockerignore file is ready only from root of context, it will never be used? It would need to be placed in all directories for specified versions, right?

Reply:
I think that depends how you call docker or have your docker-compose. it's untested now but I think you can do `docker build -f 0.7.53` and it should respect the root's .dockerignore. but probably not if you first change into the directory. in docker-compose you can also set the dockerfile's path. 

on a sidenote, is there a reason why you chose to integrate all versions in the same branch? I usually put each version in its own branch, at least in downstream projects like this where I don't want "stable" release tags. but I might overlook smth here.

Reply:
The only reason why it is done like that, that it's far easier to update all the versions at once, than to replicate the same changes across many branches. As well, after updates I usually rebuild all the versions so I'm skipping all the checkouts.

Regarding dockerignore, see (this)[https://docs.docker.com/engine/reference/builder/#dockerignore-file]:
> Before the docker CLI sends the context to the docker daemon, it looks for a file named .dockerignore in the root directory of the context. 

So to make it work, it would probably simplify also other parts, as we can then invoke it like:
```
docker build -f 0.7.53/Dockerfile .
```
And do not copy `bin/`, `docker-entrypoint.sh`, `etc` and `requirements.txt` to all the folders. Only Dockerfiles are different.

Looks good, I can take this on my own.